File: c/utime.c
#include <sys/types.h>
#include <utime.h>

int main() {
  struct utimbuf buf;
  buf.actime = 0;
  buf.modtime = 0;

  int ret = utime("utime.c", &buf);
  if (ret) {
    perror("utime");
    exit(1);
  }

  return 0;
}

File: rust-stable/src/bin/utime.rs
extern crate libc;

use std::ffi::CString;
use libc::types::os::arch::posix01::utimbuf;
use libc::funcs::posix88::unistd::utime;

fn main() {
    let buf = utimbuf {
        actime: 0,
        modtime: 0,
    };

    let name = CString::new("utime").unwrap();

    let ret = unsafe { utime(name.as_ptr(), &buf) };
    if ret != 0 {
        println!("utime: 실패!");
        std::process::exit(1);
    }
}
